OracleJava/README.md

```diff
@@ -1,23 +1,30 @@
 Oracle Java on Docker
 =====
-This repository contains sample Docker configurations to facilitate installation and environment setup for DevOps users. This project includes Dockerfiles based on Oracle Linux for all the currently available Oracle JDK's and Server JRE 8.
+This repository contains sample Docker configurations to facilitate installation and environment setup for DevOps users. This project includes Dockerfiles based on Oracle Linux with JDK images of currently supportes versions 11 and later and for Server JRE 8.
```

Comment:
This repository contains sample Docker configurations to facilitate installation and environment setup for DevOps users. This project includes Dockerfiles based on Oracle Linux with JDK images of currently **supported** versions 11 and later and for Server JRE 8.

Should we put "11 and later" in brackets()?

Reply:
Clearly "yes" on the typo fix so supportes -> supported, but I would leave the 11 and later outside of brackets unless you change the wording to something more generic.  The problem is that we really have 2 different things.  JDK for 11+ and ServerJRE for 8.  If you want it more clear we would need two paragraphs instead of one catch-all.

Reply:
Fixed typo.  Good to go?